src/main/java/de/tum/in/www1/artemis/web/rest/LearningPathResource.java

```diff
@@ -162,8 +180,12 @@ public ResponseEntity<NgxLearningPathDTO> getLearningPathNgxGraph(@PathVariable
         else if (!authorizationCheckService.isAtLeastInstructorInCourse(course, user) && !authorizationCheckService.isAdmin()) {
```

Comment:
`isAtLeastInstructorInCourse` returns true for all admins.
```suggestion
        else if (!authorizationCheckService.isAdmin()) {
```